### DISCUSSION TITLE
Welcome to BMO-Utils Discussions!

### DISCUSSION THREAD

**Discussion by orbian** [BYOBMO/BMO-Utils#1] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by HammerOfGrabthar** [comment]
First I'd like to thank you for making this available. I can't imagine the amount of hours it has taken in order to do so. Thank you. I have two boys, ages 13 and 10 who are crazy-excited about building the Big BMO.

That leads me to my question. I'm about to begin installing everything that you have made available here. I am noticing that it looks like all the instructions were made for the smaller BMO boy. 
Will everything here work for the BMO desktop model? I purchased the screen and the circuit boards directly from your store. Will I need to change any settings due to the screen being HDMI?

Thank you for your time. My boys will be making a thank you video very soon. I will link to the video here.

**Reply by orbian** [reply]
The differences between the handheld and desktop version are minimal so everything should work regardless of which version you build. The screen is the only difference. The handheld screen is 320x240 but in actuality it's 720x480 down sampled. The HDMI screen is 800x600. You might have to tweak the config.txt to set the proper NTSC mode if things don't look right though.

**Comment by DragoFire2164** [comment]
how do i set it up on a raspberry pi 3 i dont understand the writen instructions. thank you for any help

**Reply by orbian** [reply]
DO you have RetroPie installed? That is the first requirement. Everything else is built on top of that.

**Comment by konamiis** [comment]
im genuinely so confused on how you add the block to emulationstation system configs, ive tried so many times but i keep screwing up and getting the system config error on emulationstation ( ꩜ ᯅ ꩜;)⁭ ⁭

**Reply by orbian** [reply]
What is the error you are getting?

**Reply by konamiis** [reply]
I keep getting 
"It looks like your systems configuration file has not been set up or is invalid. You'll need to do this by hand, unfortunately"
I'm honestly a noobie at this so i don't know what I'm doing wrong

**Reply by orbian** [reply]
Post a copy of your modified config file

**Reply by konamiis** [reply]
i ended up restarting after the error, so i currently just have the basic config file but if you want that I can post that

**Reply by konamiis** [reply]
basically what do you do when you get to the page in the terminal after 
`sudo nano /etc/emulationstation/es_systems.cfg`
that's where i messed up.
